tests/test_ingesters.py

```diff
@@ -290,15 +290,13 @@ def test_ingest_from_metadata_string_geometry(self):
         self.assertEqual(inserted_dataset.geographic_location.geometry,
                          GEOSGeometry(('POLYGON((1 1, 1 2, 2 2, 2 1, 1 1))'), srid=4326))
 
+
 class DDXIngesterTestCase(django.test.TestCase):
     """Test the DDXIngester"""
 
     TEST_DATA = {
         'full_ddx': {
-            'url': "https://test-opendap.com/full_dataset.nc.ddx",
-            'file_path': "data/opendap/full_ddx.xml"},
-        'full_ddx_2': {
-            'url': "https://test-opendap2.com/full_dataset.nc.ddx",
+            'url': "https://opendap.jpl.nasa.gov/opendap/Afull_dataset.nc.ddx",
```

Comment:
why the `A`?

Reply:
fixed in 61740e4. It was just a name and does not do anything in coding! So there is no particular reason for doing it   :)